packages/svelte/src/compiler/phases/1-parse/read/style.js

```diff
@@ -139,6 +139,8 @@ function read_selector_list(parser, inside_pseudo_class = false) {
 
 		const end = parser.index;
 
+		allow_comment_or_whitespace(parser);
+
 		parser.allow_whitespace();
```

Comment:
Redundant AFAIK
```suggestion
```

Reply:
Ugh yeah definitely...i removed on the other one but forgot here